### PR TITLE
v1.0

### PR DESCRIPTION
**Pull request by erademacher**
- Updated to CCAPI 2023-04-10
- Added a changelog
- Updated usage instructions in the README and copied it to the root directory

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+NOTES:
+
+* The Go SDK is now generally available with semantic versioning compatibility promises.
+* Now using CC API version 2023-04-10.
+
+BREAKING CHANGES:
+
+* The following enums have been renamed, adding a "Type" suffix:
+  * ApiCloudProvider (renamed to CloudProviderType)
+  * AwsEndpointConnectionStatus
```
Comment (jenngeorge) [inline CHANGELOG.md:12]: `SetAwsEndpointConnectionState` appears in this list twice. We can remove both and instead say something like:
- `AwsEndpointConnectionStatus` 
  - `SetAwsEndpointConnectionStatusType` is a new enum that includes the settable statuses from `AwsEndpointConnectionStatusType`.

Also `CockroachCloudSetAwsEndpointConnectionStateRequest` is renamed to `SetAwsEndpointConnectionStateRequest`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  * LogLevel
+  * MetricExportStatus
+  * NetworkVisibility
+  * NodeStatus
+  * OrganizationUserRole
+  * OperatingSystem
+  * Plan
+  * PrivateEndpointServiceStatus
+  * SetAwsEndpointConnectionState
+
```
Comment (jenngeorge) [inline CHANGELOG.md:27]: - ApiDatadogSite renamed to DatadogSiteType

**Review by jenngeorge — APPROVED** [review]
LGTM after changelog comments!

**Comment by erademacher** [comment]
Fixed. TFTR!

**Pull request merged**